lib/improver/blending/weights.py

```diff
@@ -670,8 +670,12 @@ def _create_coord_and_dims_list(
 
         Returns:
             new_coord_list (list):
-                List of coordinates, which may include the exception_coord_name
-                taken from the cube_with_exception_coord.
+                List of tuples where each tuple is of the form
+                (coordinate, index_of_coord). This list, may include the
+                exception_coord_name taken from the cube_with_exception_coord.
+                If the coordinate does not have an associated dimension e.g.
+                the coordinate is a scalar coordinate, then the index of the
+                coordinate is set equal to 1.
```

Comment:
I think this should be "set equal to None"

Reply:
You're correct. I've corrected this.